decision-process/README.md

```diff
@@ -0,0 +1,77 @@
+---
+type: decision
+stakeholders:
+    - sig-architecture
+    - team-horizon
+    - team-team
+issue:
+---
+
+# RFC Decision Making Process
+
+This RFC follows on from the RFC repository README (see [here](https://github.com/giantswarm/rfc/blob/b3f4d2af6ff8965e542baae7d0e8fadb9163f356/README.md)).
+
+This RFC aims to expand on the current RFC process by adding a lightweight decision making process.
+The aim of this decision making process is to help enable, improve, and streamline technical decision making at Giant Swarm.
+
+## RFC Types
+
+To allow for a decision making process without removing the current free-form nature of RFCs, RFC types are proposed. The following changes will be introduced:
+
+- All RFCs will have a YAML header (front matter)
+- The YAML header will contain the following mandatory fields:
+    - `type`
+
+The 'default' type is `general`, and has no other requirements, to allow for free form writing. All currently merged RFCs would fall under this type, and will be moved to this type.
+
+The following is an entire and valid example of a YAML header for an RFC:
+
+```
+---
+type: general
+---
+```
+
+## Decision Making Process
+
+The two main aims of the decision making process are to ensure that the relevant groups have seen and approved the decision, and that the implementation of the decision is being tracked.
+
+To that end, the `type` field can also be `decision`. It is understood that when a `decision` RFC has been merged, that the decision has been taken and agreed on.
+
+If the `type` is `decision`, the following fields would also be mandatory:
+
+- `stakeholders`
```

Comment:
"stakeholders", similar to "owners", is a very vague word. Since the purpose of this field is described below, a clear name might be `required_approvers`. 

What's very important for RFCs are visibility and searchability. If I want to list all RFCs that relate to KaaS, that needs to be possible in two clicks. So for example, this could be `areas: [...]` or `topics: [...]` where a fixed set of topics could be chosen from (KaaS, product, sales, ...).